mkosi

```diff
@@ -1626,6 +1756,30 @@ def find_postinst_script(args):
     if os.path.exists("mkosi.postinst"):
         args.postinst_script = "mkosi.postinst"
 
+def find_passphrase(args):
+
+    if args.encrypt is None:
+        args.passphrase = None
+        return
+
+    try:
+        passphrase_mode = os.stat('mkosi.passphrase').st_mode & (stat.S_IRWXU | stat.S_IRWXG | stat.S_IRWXO)
+        if (passphrase_mode & stat.S_IRWXU > 0o600) or (passphrase_mode & (stat.S_IRWXG | stat.S_IRWXO) > 0):
+            print_step("Permissions of 'mkosi.passphrase' of '{}' are too open. Please make sure that the file is only accessible by its owner. Aborting.".format(oct(passphrase_mode)))
```

Comment:
indeed! I force pushed a new version now that rewords the message. I hope this one is better, please have a look. I also change this to use sys.stderr.write() like the rest of the code.

thanks for the review!